bootstrap/py_bootstrap.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+REM Check if Python is installed
+where python > nul 2>&1
+if %errorlevel% neq 0 (
+    echo Python is not installed
+    exit /b 1
+)
+
+REM Check if venv module is available
+python -m venv --help > nul 2>&1
+if %errorlevel% neq 0 (
+    echo venv module is not available
+    exit /b 1
+)
+
+set "SCRIPT_DIR=%~dp0"
+set "VENV_DIR=%SCRIPT_DIR%venv"
+
+if exist "%VENV_DIR%" (
+    call "%VENV_DIR%\Scripts\activate"
+) else (
+    echo Creating virtualenv
+    python -m venv "%VENV_DIR%"
+    call "%VENV_DIR%\Scripts\activate"
```

Comment:
Aren't we supposed to call activate.bat here? Why does this even work?

Reply:
Windows allows referencing files without explicit extensions in some scenarios. Made it explicit in [replace activate with activate.bat](https://github.com/apache/nifi-minifi-cpp/pull/1681/commits/833065382bd84716d239c0f2b7871ae8a66dd5da)

Reply:
I was especially confused, because there is an activate file in the same location without an extension, but it's a bash script. Thx for changing it.